main.tf.templ

```diff
@@ -29,3 +29,36 @@ module "k8s" {
     microservice_domain_name    = "$(microservice_domain_name)"
     microservice_namespace      = "$(microservice_namespace)"
 }
+
+module "gke-redis" {
+    source                      = "./redis/gke-redis"
+    project                     = "$(project)"
+    region                      = "$(region)"
+    zone                        = "$(zone)" 
```

Comment:
If I recall correctly `Terraform` throws a warning to have this changed to `location` since `zone` has been/will be deprecated.